test/formio-wrapper.test.js

```diff
@@ -623,5 +615,34 @@ describe('Formio Wrapper Tests.', () => {
     assert.notCalled(spied);
   });
 
+  it('_packageData works as expected', async () => {
+    const data1 = {
```

Comment:
Should we change the name of variables?

Reply:
This is tests, they have no actual meaning.